homeassistant/components/samsungtv/triggers/turn_off.py

```diff
@@ -0,0 +1,108 @@
+"""Samsung TV device turn off trigger."""
```

Comment:
This file is 99.7% similar to `turn_on.py`. You should reuse the code not duplicate it.

Reply:
I think I'm too dumb to do this. I got little to no Python experience. I really tried though. Would you mind doing it yourself? I can give you the needed permissions.